wiki/Tournaments/GBC/GBC_2023_Autumn/en.md

```diff
@@ -159,6 +163,183 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
 | D15 | **厕所战士** | ::{ flag=SG }:: **[oyama mahiro](https://osu.ppy.sh/users/21207265)**, ::{ flag=MY }:: [Tosai_](https://osu.ppy.sh/users/3760209), ::{ flag=ID }:: [\[LS\]Robi](https://osu.ppy.sh/users/21900872), ::{ flag=PE }:: [Sakisagee](https://osu.ppy.sh/users/9938020), ::{ flag=HK }:: [ERDFS](https://osu.ppy.sh/users/28707779) |
 | D16 | **V guys brger n’ frys** | ::{ flag=US }:: **[YoungBurger12](https://osu.ppy.sh/users/18667176)**, ::{ flag=US }:: [Surcanp](https://osu.ppy.sh/users/19862784), ::{ flag=US }:: [trooperr](https://osu.ppy.sh/users/32028459), ::{ flag=TH }:: [MickyGG](https://osu.ppy.sh/users/27010218), ::{ flag=US }:: [SolarIsCool](https://osu.ppy.sh/users/33833222) |
 
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+### QQ Division
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Q12: Team 12 |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | Q3: 兰州拉面专业培训班 |
+| ![Bronze crown片](/wiki/shared/crown-bronze.png "3rd place") | Q16: DWG.BurgerkingReisen |
+
+### Discord Division
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | D9: 麦当劳护肤 |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | D2: viets sing |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | D12: Sunaookami Shiroko |
+
+## Match results
+
+### QQ Division Grand Finals
+
+Detailed statistics for this round can be found [here](https://docs.qq.com/sheet/DTU92VHRFUm1nTm9Z?tab=xiki0i).
+
+Saturday, 16 December 2023:
+
+| Team A |  |  | Team B | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Q3: 兰州拉面专业培训班** | **7** | 3 | Q16: DWG.BurgerkingReisen | [#1](https://osu.ppy.sh/community/matches/111749143) |
+| **Q12: Team 12** | **7** | 5 | Q3: 兰州拉面专业培训班 | [#2](https://osu.ppy.sh/community/matches/111750301) |
```

Comment:
the match links should all be `#1`. it's just a formatting thing. if there are two multiplayer lobbies for one match, it'd be `[#1](), [#2]()`